### PR TITLE
Hw 6

### PR DESCRIPTION
**Pull request by Giliaz**

### CODE REVIEW

File: hw_6/context_hw6.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+class Colorized:
```
Comment (mikolaz27) [inline hw_6/context_hw6.py]: better here change to lower case

File: hw_6/context_hw6.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                       'CYAN': '\033[96m'}
+
+    def __enter__(self):
+        print(self.colors[self.key])
+        return self.key
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        Exception(exc_type)
+        self.key = 'WHITE'
+        print(self.colors[self.key], f'Printed in default colors {self.key}')
```
Comment (mikolaz27) [inline hw_6/context_hw6.py]: no needs to print in default color very run, remove

**Review by mikolaz27 — APPROVED** [review]

File: hw_6/context_hw6.py (hunk 1 above)
Comment (Giliaz) [reply hw_6/context_hw6.py]: corrected

File: hw_6/context_hw6.py (hunk 2 above)
Comment (Giliaz) [reply hw_6/context_hw6.py]: corrected

**Pull request merged**